README.md

```diff
@@ -6,6 +6,20 @@ The module has two stages "discovery" and "apply". During the "discovery" stage
 
 The configuration object should be stored within a file, then before invoking the module, read the file will be read and pass the resulting the object into the module call. This should have the effect of being able to store our active directory configuration within Git.
 
+## Requirements
+
+### RSAT-Powershell Windows Feature
+
+The UKHO.ADM module sits on top of the RSAT-Powershell functions, unfortunately this cannot be added as a Required Module as it is specific to windows 10 and server 2016. 
```

Comment:
Sorry. Can we use this instead:

The UKHO.ADM module uses RSAT-PowerShell functions. These functions are only available after installing [RSAT](https://support.microsoft.com/en-gb/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems) and cannot be downloaded from PsGallery.

#### Install RSAT on a Windows Server 

```powershell
Add-WindowsFeature RSAT-AD-PowerShell 
```

#### Install RSAT on a Windows Desktop

Install relevant version for your [Windows Desktop version](https://support.microsoft.com/en-gb/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).

Reply:
updated based on suggestions